<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clock_and_Reset_model</class>
 <widget class="QFrame" name="Clock_and_Reset_model">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="modelabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Mode :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelresetmodelname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Reset Model name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="pinplabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>PinP :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelresetnetworkname">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Reset Network name:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="resetNetwork_pinP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>400</y>
     <width>171</width>
     <height>33</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelclockmodelname_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Clock Network :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addclocknetwork">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addresetnetwork">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>500</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fpgaRef">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>251</width>
     <height>33</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="clockNetwork_pinP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>121</width>
     <height>33</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelfpgaref">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>FPGA reference :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelclockmodelname_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Clock Network :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="clockNetwork_pinN">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelclockmodelname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Clock Model name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelclockmodel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Reset Model :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="clockMode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Default</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lvds</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="pinnlabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>PinN :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="resetModelContainer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>351</width>
     <height>131</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="clockModelContainer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="modelabel_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Mode :</string>
   </property>
  </widget>
  <widget class="QLabel" name="pinplabel_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>PinP :</string>
   </property>
  </widget>
  <widget class="QLabel" name="pinnlabel_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>430</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>PinN :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="resetMode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Default</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lvds</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelclockmodelname_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Clock Network name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelclockmodel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Clock Model :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="resetNetwork_type">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>460</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ActiveLow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ActiveHigh</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="modelabel_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="resetNetwork_pinN">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>460</y>
     <width>171</width>
     <height>33</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>771</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>570</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>771</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="clearClockBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>570</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Clock Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearResetBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>570</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Reset Model</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>771</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="clockModel_name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="clockNetwork_name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="resetModel_name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="resetNetwork_name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>330</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
